### ISSUE TITLE
Cast to chromecast or TV

### ISSUE THREAD

**Issue by Selflabyrinth1** [KRTirtho/spotube#153]
This app is exactly what I was looking for. Well not really. I was trying to find an app that casts youtube or spotify music for free on a chromecast. This app meets my needs. Although if casting was added it would be my number 1 app for real.

It's a great app but it still needs this little feature to be added to make it as perfect as possible.

**Comment by potokslow** [comment]
I agree, adding cast / connect option to listen via Chromecast devices or other players would be awesome.

@KRTirtho 
Is there any plan to introduce the Spotify Connect / Cast functionality?
It's not explicitly mentioned under "TODO" or "Things that don't work" sections.

**Comment by mrjfilippo** [comment]
Hoping to revive this request. Spotify connect/Chromecast functionality without ads would be a killer feature.

**Comment by ergdevops** [comment]
++ 

Totally aligned with this feature request. Having this feature will make this app complete. Hoping developer gets enough motivation to pick this feature. 🙏

**Comment by Akruidenberg** [comment]
+1 for spotify connect!

**Comment by AbyssalKaz** [comment]
Bump, I would 100% use spotube everyday if I could cast it to my google hub

**Comment by Ham3D** [comment]
Bump

**Comment by fleamour** [comment]
Yes please!

**Comment by Necr0man** [comment]
This would be awesome!!

**Comment by NandayDev** [comment]
Bump!

**Comment by Nb-tst** [comment]
Maybe with something like this?
https://github.com/aschober/vinyl-cast

**Comment by AzurTheDragon** [comment]
Bump
It would be great to be able to cast audio on other devices

**Comment by ToshY** [comment]
Can a maintainer please clarify if this will be considered for the roadmap?

**Comment by kenjibailly** [comment]
Looking for something similar, casting to my sonos basically.

**Comment by ghaspias** [comment]
Besides casting using the most common (but closed and hardware dependent) Chromecast or AirPlay (like e.g. monochrome.tf), this application should implement open-source casting solutions, like FCast.
While Chromecast and/or AirPlay might be easier to implement and more well known, supporting FCast would help break the duopoly of Google (receiver hardware) and Apple (sender hardware) and allow casting from any device/platform to any (custom) receiver.
Alternatively, some kind of integration with mpd, snapcast, or other open standards (upnp/DLNA) would also be welcome.
